pkg/extensions/extension-scrub-disabled.go

```diff
@@ -0,0 +1,19 @@
+//go:build !scrub
+// +build !scrub
+
+package extensions
+
+import (
+	"zotregistry.io/zot/pkg/api/config"
+	"zotregistry.io/zot/pkg/log"
+	"zotregistry.io/zot/pkg/storage"
+)
+
+// EnableScrubExtension ...
+func EnableScrubExtension(config *config.Config,
+	storeController storage.StoreController,
+	log log.Logger,
+) {
+	log.Warn().Msg("skipping enabling scrub extension because given zot binary doesn't support any extensions," +
```

Comment:
I wanted the log message to explicitly indicate what and where is the problem, that's why I name the extension and the action that failed due to the lack of it.
What about "skipping enabling scrub extension because given zot binary doesn't include this feature, please build a binary that does so"

Reply:
Sounds good